src/player.js

```diff
@@ -844,7 +844,9 @@ export default class Player extends FakeEventTarget {
    * @returns {Array<number>} - The possible playback speeds speed of the video.
    */
   get playbackRates(): Array<number> {
-    if (this._engine) {
+    if (this._playbackRates) {
```

Comment:
need to define `_playbackRates: Array<numbers>`

Reply:
done